src/modules/ims_registrar_pcscf/save.c

```diff
@@ -196,10 +196,24 @@ static inline int update_contacts(struct sip_msg *req,struct sip_msg *rpl, udoma
 							ci.searchflag = SEARCH_RECEIVED;
 						}
 					}
-				} 
+				} else {
+					char* srcip = NULL;
+					if((srcip = pkg_malloc(50)) == NULL) {
```

Comment:
It is necessary to allocate new pkg memory here? The save_pending function just uses a char[50] array for a somewhat similar logic. Just wondered. :-)